Makefile

```diff
@@ -245,6 +245,7 @@ shared_js_files=$(wildcard shared/js/*.js)
 shared_jsx_files=$(wildcard shared/js/*.jsx)
```

Comment:
Can get rid of the comment a few lines above here: `The shared files currently get built once for each destination; we could optimize this away. We use the shared_jsx_files to make this more readable.`